drivers/at/at.c

```diff
@@ -25,11 +25,11 @@
 #endif
 
 #if defined(MODULE_AT_URC_ISR_LOW)
```

Comment:
This should then also be called `MODULE_AT_URC_ISR_LOWEST` (even though `at_urc_isr_low` *does* exist).

Reply:
Alright, I'll change it real quick